clang/lib/Parse/ParseDecl.cpp

```diff
@@ -331,6 +331,13 @@ unsigned Parser::ParseAttributeArgsCommon(
     IdentifierInfo *AttrName, SourceLocation AttrNameLoc,
     ParsedAttributes &Attrs, SourceLocation *EndLoc, IdentifierInfo *ScopeName,
     SourceLocation ScopeLoc, ParsedAttr::Syntax Syntax) {
+
+  if (attributeIsTypeArgAttr(*AttrName)) {
+    ParseAttributeWithTypeArg(*AttrName, AttrNameLoc, Attrs, EndLoc, ScopeName,
+                              ScopeLoc, Syntax);
+    return 1;
```

Comment:
Is this the number of arguments? Can we be sure all of the type argument attributes have only one argument? Maybe we should at least leave a FIXME.